platform/spire/src/setup.py

```diff
@@ -228,7 +228,9 @@ def modify_dns_bootstrap(ops: Operations, is_install: bool) -> None:
 
 def dns_bootstrap_lines() -> str:
     config = configuration.get_config()
-    return "".join("%s\t%s # AUTO-HOMEWORLD-BOOTSTRAP\n" % (ip, hostname) for hostname, ip in config.dns_bootstrap.items())
+    lines = [(ip, hostname) for hostname, ip in config.dns_bootstrap.items()]
+    lines += [(node.ip, node.hostname) for node in config.nodes]
+    return "".join("%s\t%s # AUTO-HOMEWORLD-BOOTSTRAP\n" % (line[0], line[1]) for line in lines)
```

Comment:
Couple optional style comments:
 - Might be clearer if you use a more specific variable name than `lines`
 - Might be clearer to do ` for ip, hostname in lines` than ` for line in lines` and have to index into each line.